### PR TITLE
[New] MySQL one-click

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Created a new guide for MySQL/MariaDB one-click app.

PS: I have retained the links to external docs to MySQL docs

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit a756fa9d039a0a3a35af53a1aebfbfea398ab21b

https://deploy-preview-3199--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 9dc5b2faaa00d8d90eb6d0a2dba6b178d89b580a

https://deploy-preview-3199--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by Rajakavitha1** [comment]
Here's the preview of the guide: https://deploy-preview-3199--nostalgic-ptolemy-b01ab8.netlify.com/platform/one-click/deploy-mysql-with-one-click-apps/

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+{{< content deploy-one-click-apps>}}
+
+### MySQL/MariaDB Options
+
+| **Field** | **Description** |
+|:--------------|:------------|
+| **MySQL User** | The user for your MySQLDB database. *Required*. |
+| **MySQL Root Password** | The root password for your MySQL database. *Required*. |
+| **Create Database** | The database on your MySQL . *Required*. |
```
Comment (andystevensname) [inline docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: Remove space before period.

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+| **Region** | The region where you would like your Linode to reside. In general, it's best to choose a location that's closest to you. For more information on choosing a DC, review the [How to Choose a Data Center](/docs/platform/how-to-choose-a-data-center) guide. You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network routes between you and each of our data centers. *Required*. |
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions). The Linode plan you deploy your MERN stack on should account for the estimated workload. If you are standing up a simple web page, you can use a Nanode or 2GB Linode. If you will deploy a more robust web app, then consider a plan with higher RAM and CPU allocations. If you decide that you need more or fewer hardware resources after you deploy your app, you can always [resize your Linode](/docs/platform/disk-images/resizing-a-linode/) to a different plan. *Required*. |
```
Comment (andystevensname) [inline docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: Change copy to reflect MySQL app, not MERN stack.

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        use       (\u) Use another database. Takes database name as argument.
+        charset   (\C) Switch to another charset. Might be needed for processing binlog with multi-byte charsets.
+        warnings  (\W) Show warnings after every statement.
+        nowarning (\w) Don't show warnings after every statement.
+
+        For server side help, type 'help contents'
+
+        MariaDB [(none)]>
+
+3.  Grant access to the database that you created when launching the One-Click App for **MySQL User** . In this example, the database is called `webdata`, the user `webuser`, and password of the user is `password`. Be sure to enter your own password. This should be different from the root password for MySQL:
```
Comment (andystevensname) [inline docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: Remove space after "MySQL User."

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md (hunk 1 above)
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: fixed.

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md (hunk 2 above)
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: fixed.

File: docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md (hunk 3 above)
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md]: fixed

**Pull request merged**